src/shared/components/Cart/component.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import { If } from 'react-if';
+import { H3, Text } from 'shared/Typography';
+import { Oval, ParallelogramOvalToLeft, RectangleOvalToRight } from 'shared/shapes';
+import { CartWrapper, ImageWrapper, CartContent, CartSubHeader } from './styles';
+
+type CartProps = {
+  imgName?: string;
+  imgURL: string;
+  caption: string;
+  title: string;
+  text?: string;
+  size: 'small' | 'medium' | 'large';
+  textColor?: string;
```

Comment:
should be `textColor?: Colors` - and let's pass Colors values

Reply:
Done